assets/storageclass_tier3.yaml

```diff
@@ -2,6 +2,8 @@ kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
   name: tier3-csi
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
```

Comment:
It will be good if we can hear more from others as well on this option, like `T3` vs `T1` as default